models/classifieds.js

```diff
@@ -0,0 +1,53 @@
+module.exports = function(sequelize, Sequelize) {
+  const classifieds = sequelize.define('classifieds', {
+    id: {
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    author_id: {
+      type: Sequelize.STRING,
+      notEmpty: true
+    },
+    username: {
+      type: Sequelize.TEXT
+    },
+    email: {
+      type: Sequelize.STRING,
+      validate: {
+        isEmail: true
+      }
+    },
+    title: {
+      type: Sequelize.STRING,
+      notEmpty: true,
+      validate: {
+        len: [1,50]
+      }
+    },
+    price: {
+      type: Sequelize.INTEGER,
+      notEmpty: true,
```

Comment:
@JoeWrightNC  does this still work even with this trailing comma? i forgot if trailing commas break in sequelize methods, as they would in a json file for example.

Reply:
@JoeWrightNC  also, don't forget to switch the branch to develop. it will default to master so we manually change it everytime.